frontend/components/forms/fields/InputFieldHiddenContent/InputFieldHiddenContent.tsx

```diff
@@ -14,76 +10,31 @@ interface IInputFieldHiddenContentProps {
   value: string;
   name?: string;
   className?: string;
+  helpText?: string | JSX.Element;
 }
 
+/** Used to easily create an InputField with a show/hide and copy buttion */
 const InputFieldHiddenContent = ({
   value,
   name,
   className,
+  helpText,
 }: IInputFieldHiddenContentProps) => {
-  const [copyMessage, setCopyMessage] = useState("");
-  const [showSecret, setShowSecret] = useState(false);
-
   const classNames = classnames(baseClass, className);
 
-  const onCopySecret = (evt: React.MouseEvent) => {
```

Comment:
Moved this onto InputField so we could incorporate textarea copy as used in Enroll Secrets modal